### DISCUSSION TITLE
Android 12 Go edition

### DISCUSSION THREAD

**Discussion by jhkaiser** [LinwoodDev/Butterfly#678] [General]
I wonder whether the app runs on Android 12 Go edition.  My initial impression is NO.  To start with, the main view does not show the save button.  Then, all things I try to add seem to disappear immediately after switching to another tool. Specifically, creation of an area works or works not and cannot be edited afterwards.

**Comment by CodeDoctorDE** [comment]
Hmm it should just work fine on it.
The main view has no save button because it autosaves. What things disappear? Can you send me a video showing it?
Areas can be created with the area tool and edited by using double click

**Reply by jhkaiser** [reply]
Thanks for your prompt reply.  Things start to work.  The crucial tip was working with double-click.  In addition, I omitted to use areas -- I still do not come to terms with them.  Initially, I thought that I need them to add any content to the note but I found that I can do so without.  On small thing to note is that one-liner labels do not show the move/rotate marks after caught by the lasso.  So I move them by cut-and-paste.

**Reply by CodeDoctorDE** [reply]
Areas are if you need a standard pages system like if you want to export to pdf.
The one liner text element is too small to display these marks. I'm currently not sure how I can improve this. If I display it, it overlaps.
